commit 1a81ee5c8406d2d134807021f32e679d914d74cf (refs/changes/16/4616/1)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2018-08-01T14:59:31-04:00 (1 year, 2 months ago)
    
    OS-7009 varpd fails to start on boot, missing overlay module

diff --git a/usr/src/cmd/zoneadm/zones.xml b/usr/src/cmd/zoneadm/zones.xml
index 1b6f8cd49b..b094bc660b 100644
--- a/usr/src/cmd/zoneadm/zones.xml
+++ b/usr/src/cmd/zoneadm/zones.xml
@@ -54,6 +54,19 @@
 		<service_fmri value='svc:/milestone/multi-user-server' />
 	</dependency>
 
+	<!--
+	    Until overlay device creation is moved out of the zone
+	    state-change script, zones must be dependent on varpd's
+	    successful launch.
+	-->
+	<dependency
+		name='varpd'
+		type='service'
+		grouping='require_all'
+		restart_on='none'>
+		<service_fmri value='svc:/network/varpd' />
+	</dependency>
+
 	<dependency
 		name='metadata'
 		type='service'
